src/commands/remove.js

```diff
@@ -7,20 +7,34 @@ const {userConfig, isLoggedIn, logout} = require('../config');
 
 exports.command = ['remove <id>', 'rm <id>'];
 exports.describe = 'remove active deployment';
-exports.builder = {};
-exports.handler = async ({id}) => {
-  if (!isLoggedIn()) {
+exports.builder = {
+  token: {
+    alias: 't',
+    description: 'Deployment token to be used for authentication',
+  },
+};
+exports.handler = async (args = {}) => {
+  const deployToken = args.token;
+  const id = args.id;
+
+  if (!deployToken && !isLoggedIn()) {
     return;
```

Comment:
There should be an error message here notifying user that he needs to auth first.

Reply:
I used the same logic, deploy.js is using... Im adding this :)

Reply:
Does `deploy.js` also not have an error message? 😮
If yes, I'd need to fix this 🤔

Reply:
Yeah, it doesnt. There you go, there is now a error message :)